gui/GameConqueror.py

```diff
@@ -1040,9 +1039,26 @@ def check_backend_version(self):
         if self.backend.get_version() != VERSION:
             self.show_error(_('Version of scanmem mismatched, you may encounter problems. Please make sure you are using the same version of GameConqueror as scanmem.'))
 
-    
 
 if __name__ == '__main__':
+    # Parse cmdline arguments
+    parser = argparse.ArgumentParser(prog='GameConqueror',
+                                     description="A GUI for scanmem, a game hacking tool"
+
+                                    )
```

Comment:
I introduced this by error when rebasing, see 1866e11f522eb2de002eca6e54543bd04248347c .